### PR TITLE
Fix for Stram web service calls failing in secure non-HA mode

### PR DESCRIPTION
**Pull request by pramodin**
Fix for property name used to lookup RM webapp address in non-HA mode and additional log statements to help with debugging the request processing in future.

### CODE REVIEW

**Comment by pramodin** [comment]
Review @tweise or @davidyan74

File: engine/src/main/java/com/datatorrent/stram/security/StramWSFilter.java [hunk 1]
```diff
@@ -99,6 +99,7 @@ public void init(FilterConfig conf) throws ServletException
         proxyAddresses = new HashSet<String>();
         for (String proxyHost : proxyHosts) {
           try {
+            logger.debug("resolving proxy hostname {}", proxyHost);
```
Comment (tweise) [inline engine/src/main/java/com/datatorrent/stram/security/StramWSFilter.java:103]: Is this logging that occurs on every request? If so, we should take it out.

Comment (pramodin) [inline engine/src/main/java/com/datatorrent/stram/security/StramWSFilter.java:103]: Not every request but periodically (5mins) and its in debug so normally it won't be logged unless asked for.

**Comment by pramodin** [comment]
Updated logging with more detailed messages and more strategic placement for them.

**Comment by tweise** [comment]
Please submit a separate pull request for devel-3. I was not able to cherry-pick as the change introduces a checkstyle violation.

**Pull request merged**